### DISCUSSION TITLE
Digital products

### DISCUSSION THREAD

**Discussion by AmineYagoub** [vendurehq/vendure#1894] [Q&A] [answered]
Vendure comes with pre-existing widgets and plugins but I c'ant find any one suitable for digital products.
if Istarted new project with vendure to build gaming store , do I need to build custom plugin or vendure already support selling digital products Out-of-the-box?

**Comment by michaelbromley** [comment, marked as answer]
Hi,
Support for digital products comes down to 2 aspects:

1. Shipping: digital goods should have a shipping cost of 0.
2. Fulfillment: digitals goods aren't physically "shipped" but are nonetheless still "fulfilled", which might be by sending an email to the customer with a code/link, etc.

Take a look at the [Shipping Guide](https://www.vendure.io/docs/developer-guide/shipping/) for how to build custom implementations for these.
